server/app/api/v1/endpoints/webhooks.py

```
@@ -214,4 +214,130 @@ async def stripe_webhook(
+                )
+                call = result.scalar_one_or_none()
+                
+                if call and call.rooftop:
+                    rooftop = call.rooftop
+                    if call.service_record and call.service_record.customer_name:
+                        customer_name = call.service_record.customer_name
+                        
+                    logger.info(
+                        f"📝 [VAPI-FUNC] Found call {call_id} for rooftop: {rooftop.name}"
+                    )
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)